test/documentation/seqan3_doxygen_cfg.in

```diff
@@ -1,11 +1,15 @@
 PROJECT_NAME           = "SeqAn3"
+PROJECT_BRIEF          = "The Modern C++ library for Sequence Analysis in Bioinformatics."
+PROJECT_LOGO           =
+LAYOUT_FILE            = ${SEQAN3_DOXYGEN_SOURCE_DIR}/test/documentation/DoxygenLayout.xml
```

Comment:
```suggestion
LAYOUT_FILE            = ${SEQAN3_DOXYGEN_SOURCE_DIR}/test/documentation/DoxygenLayout.xml}
```
?

Reply:
to me this seems fine – why the additional bracket in the end? (@smehringer)

Reply:
ops